source/playerTurn.py

```diff
@@ -0,0 +1,7 @@
+def playerTurn(players, current_player, getCard):
+    num_players = len(players)
+    if getCard:
+        return current_player
+    else:
+        next_player = (current_player + 1) % num_players
+        return next_player 
```

Comment:
nice and simple, makes sense easy to follow

Reply:
Thanks for reviewing Robert.